src/client.ts

```diff
@@ -177,6 +189,7 @@ class Client {
   public publish(publication: Publication): void {
     this._sendToHost({
       msgType: 'publish',
+      clientId: this._clientId,
```

Comment:
I will add a similar id to my worker bootstrap lifecycle method.

@MattCheely and I were discussing this.  This will allow the host app to make decisions on priority and honoring the request (nav, toast, etc.) based on what client is requesting and what's currently running.